lib/route/route.js

```diff
@@ -354,9 +358,65 @@ const checkNextInputPage = ({ _id, params }, userData) => {
   }
 }
 
-const getNextPage = (currentPage, userData, recurse) => {
+function getNextPage (currentPage, userData, recurse) {
+  const { _id } = currentPage
+  const currentInstance = getInstance(_id)
+
+  /*
+   *  If this page has upload components, ensure that the user proceeds to the check step
+   */
+  if (hasUploadComponents(currentInstance)) {
+    const steps = [
```

Comment:
I'm not certain this is necessary, but since I'm shifting items out of the array I thought it was better to create a copy of the array first. But I need to check whether the array I'm copying is itself a copy

Also, I could have written `getInstanceProperty(_id, 'steps', []).slice()` but I thought this was clearer